homeassistant/components/plant/__init__.py

```diff
@@ -183,6 +185,12 @@ def __init__(self, name, config):
             self._conf_check_days = self._config[CONF_CHECK_DAYS]
         self._brightness_history = DailyHistory(self._conf_check_days)
 
+        for _, reading in self.READINGS.items():
```

Comment:
Since the keys are not used you can use `.values()`:
```suggestion
        for reading in self.READINGS.values():
```

Reply:
@springstan thx; Have fixed it.